geronimo-javamail_1.6/tck.adoc

```diff
@@ -109,3 +117,6 @@ SMTP_TO=$SMTP_USERNAME
 
 From $TCKHOME, run `ant gui`.
 
+From the UI, `File->Open->TestSuite` and select the `tests` folders
```

Comment:
This is how I tried to execute from the GUI the tests.
After selecting Run the GUI complained that I was missing a work directory, I don't know if that directory comes already in the mail TCK, so I selected create one and named `JTWork` but after that I got another screen `Start the configuration editor` that is not need it according to section `4.7.3 To Configure the JavaTest Harness to Run the TCK Tests ` from the `Jakarta-Mail-TCK-Users-Guide.pdf` but in my setup I can't move forward if I don't create a configuration. 

By the end of my day this was the result :/ 
![image](https://user-images.githubusercontent.com/1454523/89606122-774e4400-d82c-11ea-9afc-bdfd2b08c8c8.png)

Reply:
This was resolved and I added to this PR a second commit with the instruction to avoid this issue.